library/ssl_debug_helpers.h

```diff
@@ -33,6 +33,11 @@
 
 const char *mbedtls_ssl_states_str( mbedtls_ssl_states in );
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+const char *mbedtls_ssl_ticket_flags_str( mbedtls_ssl_ticket_flags in );
```

Comment:
```suggestion
const char *mbedtls_ssl_tls13_ticket_flags_str( mbedtls_ssl_ticket_flags in );
```

Reply:
I am not sure whether you have gone through the code or not. 
If you have not gone through your comments or the code, your comments will change back and forth, that's confusing... see your comment https://github.com/Mbed-TLS/mbedtls/pull/6486#discussion_r1017341895

Reply:
I mean If we remove the enum structure as what you comment in https://github.com/Mbed-TLS/mbedtls/pull/6486#discussion_r1017341895  then this function will not be exist.

Reply:
Is it needed in this PR? Please read the generate script and understand the usage of `ticket_flags` before introduce it.

Reply:
If we didn't write them as **enum {}mbedtls_ssl_ticket**,  we have no need of it.
Currently, we defiine the maros seperately, so we don't need it any more.